src/components/pages/MapPage/index.js

```diff
@@ -1,39 +1,85 @@
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect, useRef } from "react";
 
 import GoogleMapReact from "google-map-react";
 import styled from "styled-components";
 
 import { getFeedInfo } from "../../../api";
 import theme from "../../../theme/theme";
-import { Icon } from "../../atoms";
+import { Icon, GpsIcon } from "../../atoms";
 import { MapTemplate } from "../../templates";
 
+import useSocket from "../../../hooks/useSocket";
+
 const StyledIcon = styled(Icon)`
   color: ${(props) => props.color};
 `;
 
 function MapPage() {
+  function getRandom() {
+    const rand09 = Math.floor(Math.random() * 10);
+    return rand09;
+  }
+
+  const random = `0.000${getRandom()}`;
+  const [socket, disconnect] = useSocket("map");
   const [defaultProps, setDefaultProps] = useState(null);
   const [zoomLevel, setZoomLevel] = useState(14);
   const [boundary, setBoundary] = useState({});
   const [feedLocation, setFeedLocation] = useState([]);
+  // const userGps = useRef();
+  const [userGps, setUserGps] = useState();
+  const [gpsLocations, setgpsLocations] = useState({});
```

Comment:
```suggestion
  const [gpsLocations, setGpsLocations] = useState({});
```
이렇게 네이밍 할 수도 있을 것 같습니다.👍